containers/verifier/install-dependencies.sh.j2

```diff
@@ -70,10 +76,6 @@ chmod 555 /usr/local/bin/su-exec
 su-exec nobody true
 
 # Python packages needed by Golem's imgverifier and Concent's verifier
-python3.6 -m pip install --upgrade pip
-python3.6 -m pip install --no-cache-dir --requirement /golem/work/imgverifier-requirements.txt
-
-# Clean up
-apt-get clean
-apt autoremove --assume-yes
-rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* ~/.cache/pip/*
```

Comment:
This needs to be done inside the script. Otherwise it'll remain in the image.

Reply:
No, this one's different. `Dockerfile` runs `install-dependencies.sh` directly which means that once it ends, everything that was not removed by it remains in the image layer that the `RUN` command creates.

If you don't uninstall build dependencies here, they'll remain in the image.